lib/grape/path.rb

```diff
@@ -22,11 +22,19 @@ def root_prefix
     end
 
     def uses_specific_format?
-      !!(settings[:format] && Array(settings[:content_types]).size == 1)
+      if settings.key?(:format) && settings.key?(:content_types)
```

Comment:
Figured out why it was failing here and how to implement it properly.
Added a check for the presence of the keys, if either is `nil`, it will simply move on to the `else` block